docs/cody/clients/install-jetbrains.mdx

```diff
@@ -97,17 +93,59 @@ In addition, Autocomplete suggestions use the same color as inline parameter hin
 
 ## Commands
 
-Cody with JetBrains offers quick, ready-to-use Commands for common actions to write, describe, fix, and smell code. These allow you to run predefined actions with smart context-fetching anywhere in the editor, like:
+Cody with JetBrains offers quick, ready-to-use Commands for common actions to write, describe, fix, and smell code. These allow you to run predefined actions with smart context-fetching anywhere in the editor.
+
+These are broadly categorized into the Edit Commands and Chat Commands.
+
+- **Edit Code**: Makes inline code edits. You also get the option to select LLM for edit suggestions
+- **Document Code**: Create inline documentation for your code
+- **Generate Unit Test**: Creates inline unit tests for your code-
+- **Smell Code**: Finds code smells in your file
+- **Explain Code**: Expains code for your file
+
+![jetbrains-commands](https://storage.googleapis.com/sourcegraph-assets/Docs/jb-commands-new-ga.png)
 
-- Explain Selected Code
-- Generate Unit Test
-- Smell Code
+Let's learn about how to use some of these commands:
 
-![jetbrains-commands](https://storage.googleapis.com/sourcegraph-assets/Docs/jb-commands.png)
+### Inline code edits
+
+You can make edits to your code directly in your file without opening the chat window. The **Edit Code** command makes direct code edits, refactors, or bug fixes.
+
+You can run the inline edit command either on the selected code snippet or the entire file. To do so, press `Shift + Ctrl + Enter`. This opens a floating editor where you can describe the change you want to make.
```

Comment:
You can run the inline edit command on a selected code snippet, an entire file, or to generate code on a new line. To do so, use the "Edit Code" command in the Cody sidebar or context menu, or the `Shift + Ctrl + Enter` shortcut. This opens . . .